DistributionPackages/Neos.NeosIo/Resources/Private/Fusion/Components/Button.fusion

```diff
@@ -0,0 +1,15 @@
+prototype(Neos.NeosIo:Component.Button) < prototype(Neos.Fusion:Component) {
+    color = ''
+    uri = null
+
+    additionalClassName = ''
+    className = 'btn'
+    className.@process.color = ${value + (this.color ? ' ' + this.color : '' )}
+    className.@process.additionalClassName = ${value + (this.additionalClassName ? ' ' + this.additionalClassName : '' )}
+
+    renderer = afx`
+        <a class={props.className} href={props.uri}>
+            <div class="btn__content">{props.content}</div>
+        </a>
+    `
+}
```

Comment:
I would pass the classes as an array and not process it

Reply:
Guess for performance reasons